backend/modules/pdf_processing/router.py

```
@@ -1 +1,295 @@
+    """
+    logger.info("[ENDPOINT] Starting fast autofill processing...")
+    
+    if not settings.llamaparse_api_key:
+        raise HTTPException(status_code=500, detail="LlamaParse API key not configured.")
+    
+    try:
+        # Create pipeline and process
+        pipeline = get_pipeline(db, current_user)
+        result = await pipeline.process_fast_autofill(file)
+        
+        return result
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)